src/selectors/index.js

```diff
@@ -46,7 +46,12 @@ const defaultRequest = Immutable({
 })
 
 // Store dataKey-specific selectors created on demand
-const selectByKey = { data: {}, obj: {}, owr: {}, ref: {}, req: {} }
+const initialSelectorState = { data: {}, obj: {}, owr: {}, ref: {}, req: {} }
+let selectByKey = JSON.parse(JSON.stringify(initialSelectorState))
```

Comment:
I don't think we really need to clone the initial state. Can we just use a `let` binding and reassign to the same initial value inside `purgeSelectorCache`?

Reply:
thy will be done